jobs/delorean/jenkinsfiles/discovery/rc/github/Jenkinsfile

```diff
@@ -70,18 +70,43 @@ String getProductVersionID(productName) {
     return result
 }
 
-String getLatestPreRelease(projectOrg, projectRepo, githubToken, productName) {
-    //ToDo Implement this to return the latest pre release (RC) that is available, if any
+String getThreescalePrerelease(projectOrg, projectRepo, githubToken) {
+    def ghUtils = new org.integr8ly.GitHubUtils()
+    def rcTagRef = '-CR'
+    return ghUtils.ghGetLatestReleaseByTag(projectOrg, projectRepo, githubToken, rcTagRef)
+}
+
+String getAmqOnlinePrerelease(projectOrg, projectRepo, githubCredentialsID) {
+    def amqOnlineGitUrl = "git@github.com:${projectOrg}/${projectRepo}.git"
+    def amqOnlineMasterBranch = "amq-online-1-dev"
+    def amqOnlineLatestRC = ""
+    def rcFilter = "grep 'amq-online-' | grep -v 'dev'"
+    
+    dir('amq_online') {
+        gitCheckoutRepo(amqOnlineGitUrl, amqOnlineMasterBranch, githubCredentialsID, '.')
+        amqOnlineLatestRC = sh(returnStdout: true, script: "git for-each-ref refs/remotes/ --sort=-creatordate --format='%(refname:short)' | ${rcFilter} | head -n 1").trim()
+        amqOnlineLatestRC = amqOnlineLatestRC.minus("origin/")
+    }
+
+    return amqOnlineLatestRC
+}
+
+String getFuseOnlinePrerelease(projectOrg, projectRepo, githubToken) {
+    def ghUtils = new org.integr8ly.GitHubUtils()
+    return ghUtils.ghGetLatestReleaseByTag(projectOrg, projectRepo, githubToken)
```

Comment:
Not important to change now, but might be worth considering making this method static in the library